src/lightbox.js

```diff
@@ -3,6 +3,7 @@ var Helpers = require('./helpers');
 var PostMessage = require('./postmessage');
 
 module.exports = (function () {
+    var isOpened = false;
```

Comment:
why we have this variable? I see we early added one on top app level and we tracking lightbox opening